src/coreComponents/constitutive/fluid/multifluid/reactive/ReactiveFluidDriver.cpp

```diff
@@ -219,7 +219,7 @@ void ReactiveFluidDriver::runTest( FLUID_TYPE & fluid, arrayView2d< real64 > con
   }
 
   TableFunction const * waterDensityTable =
-    constitutive::PVTProps::PureWaterProperties::makeSaturationDensityTable( "helpTable", FunctionManager::getInstance() );
+    constitutive::PVTProps::PureWaterProperties::makeSaturationDensityTable( "helpTable", FunctionManager::getInstance(), false );
```

Comment:
Isn't it useful to output `PureWaterTable`. IMO, we might want to even if it is always the same and we now it is right. Just out of keeping consistency among the calls with higher `logLevel`